packages/account/package.json

```diff
@@ -31,7 +31,7 @@
     "dependencies": {
         "@binary-com/binary-document-uploader": "^2.4.8",
         "@deriv-com/analytics": "1.11.0",
-        "@deriv-com/translations": "1.3.5",
+        "@deriv-com/translations": "1.3.9",
```

Comment:
@yauheni-deriv since we are updating from 1.3.5 to 1.3.9  can we check if there was any breaking changes,
especially in regards to this discussion [https://deriv-group.slack.com/archives/C3AHMSM3P/p1722927457863969]( https://deriv-group.slack.com/archives/C3AHMSM3P/p1722927457863969)

Reply:
having done a few checks I see it is working fine. I added the note for QA team to make a sanity check for translation, @amina-deriv